### DISCUSSION TITLE
Publish docker image to google cloud registry?

### DISCUSSION THREAD

**Discussion by theganyo** [goreleaser/goreleaser#1690] [Q&A] [answered]
I'm trying to figure out how to get goreleaser to publish a docker image to a gcr.io registry, but I'm not finding any information on specifying a certain registry or authorization for it. Am I just missing something (I hope)?

**Comment by caarlos0** [comment, marked as answer]
hey, you can authorize before running goreleaser (via assume role or any other way) and just set up the docker pipe to push to gcr...

**Reply by theganyo** [reply]
That makes sense. Unfortunately, I don't have docker installed on that image, but I'll find another way to handle this. Thanks!

**Comment by hellt** [comment]
@caarlos0 and if we are talking about using goreleaser inside gh-actions or any other CI that means that before running goreleaser we need to `docker login` first and then goreleaser will be able to push the images to dockerhub?

**Reply by caarlos0** [reply]
you are correct! You can check [this](https://github.com/goreleaser/goreleaser/blob/master/.github/workflows/build.yml#L56) as an example :)
